ajax/libs/spritejs/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "spritejs",
+  "version": "3.2.8",
+  "description": "A cross platform high-performance graphics system.",
+  "filename": "spritejs.js",
```

```suggestion
  "filename": "spritejs.min.js",
```
As a minified file is available in the package, we should suggest it by defualt